package.json

```diff
@@ -10,7 +10,9 @@
   "scripts": {
     "watch": "grunt watch",
     "build:sass": "grunt compile",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "build": "npm run build:sass",
+    "lint:sass": "stylelint 'docroot/themes/humsci/**/{scss,src,color}/**/*.{scss,sass}' --config .stylelintrc",
```

Comment:
Hey @pookmish, this PR adds a task for linting the whole codebase, to the root package.json of this project. We want this to be available and useful to you when you're doing Sass development.

Once packages are installed (`npm install`) you can run the linting with `npm lint:sass` (or `npm test`). Is this the format you'd want this in? Alternatively, we could do this with a grunt task. Let me know what you think is best. 👍

Reply:
Mike confirmed with me that either works for him. 👍